browser/src/core/LOUtil.js

```diff
@@ -118,6 +118,21 @@ L.LOUtil = {
 
 		map.on('themechanged', setupIcon, this);
 	},
+	setUserImage: function(img, map) {
+		var docType = map.getDocType();
+		// set avatar image if it exist in user extract info
+		var defaultImage = L.LOUtil.getImageURL('user.svg', docType);
+		var myViewId = map._docLayer._viewId;
+		var extraInfo = map._viewInfo[myViewId].userextrainfo;
+		if (extraInfo !== undefined && extraInfo.avatar !== undefined) {
+			// set user avatar
+			img.src = extraInfo.avatar;
+			img.addEventListener('error', () => (img.src = defaultImage));
+			return;
+		}
+		img.src = defaultImage;
+		this.checkIfImageExists(img, true);
+	},
```

Comment:
I notice that there's nothing for the 'themechanged' event here, I'm undecided as to whether we should add it (we don't currently have a light/dark mode avatar difference) so I definitely don't want to block on it, but I would like to flag it to make sure it's not an oversight